### PR TITLE
Gemfile for bundler

### PR DESCRIPTION
**Pull request by Jeeppler**
This repository depends on jekyll, json and a jekyll plugin. Gemfiles are used by [Bundler](http://bundler.io/) to manage ruby dependencies. 

Issue the following commands to make use of bundler for `qubesos.github.io`
~~~
gem install bundler
cd qubesos.github.io
bundle install
bundle exec jekyll serve
~~~

`bundle exec jekyll serve` executes the command in the context of qubesos.github.io. This is helpful if one want's to fix the Jekyll version or uses another Jekyll version on the system for another project.

### CODE REVIEW

**Comment by Jeeppler** [comment]
I updated the Gemfile according to: https://help.github.com/articles/adding-jekyll-plugins-to-a-github-pages-site/ and https://github.com/github/pages-gem

**Comment by andrewdavidwong** [comment]
Sorry, I'm not familiar with this, and after reading your links, I'm still not clear on what exactly this is for. Is this just for people who want to set up a local environment for building/serving the site on their local machine? Or is this meant to affect the way the site is served by GitHub?

**Comment by Jeeppler** [comment]
The thing is installing simply jekyll is not enough you also have to install  jekyll-redirect and the json gem. All of those depenencies are bundled in the GitHub pages gem. 

Installing the GitHub pages gem resolves all the dependency issues. Bundler does not even require that you know the gem name or all the dependencies. One developer can just create a Gemfile and everybody can just run `bundle install` bundler will download and install all the dependencies for you. 

Using bundler to start jekyll allows to run jekyll with all the required jekyll plugins, even if you use other jekyll versions or gem's for other projects. Bundler will load exactly the required plugins/dependencies for this project. 

Basically the GitHub pages gem and the Gemfile make sure you have exactly the same environment as the GitHub Pages server.

**Comment by Jeeppler** [comment]
You do not have to use bundler, but it makes the life a little bit easier.
